docs/docs/reference/cloud/gatsby-cloud-environment-variables.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Environment Variables Specific to Gatsby Cloud"
```

Comment:
This and other reference guides should follow https://www.gatsbyjs.com/contributing/docs-contributions/how-to-write-a-reference-guide/

Reply:
I think you're right. However our first pass is to get the content migrated, and perform some corrections for content accuracy where warranted.